### DISCUSSION TITLE
Can the user opt out of telemetry in v2.1.0?

### DISCUSSION THREAD

**Discussion by antermin** [edrlab/thorium-reader#1738] [Ideas]
In the release notes of v2.1.0:
> * Added basic "telemetry" feature: anonymous system information sent to EDRLab server on application startup.

Can the user opt out of telemetry?

Even though it is "anonymous system information", I am still concerned about other information (e.g. computer IP address) when the application makes a connection to the telemetry server.

**Comment by danielweck** [comment]
hello, in Thorium v2.1.0 there is no "opt out" setting for telemetry. The code is open source, so the following assertion can be verified: no personal information is used such that it would allow users to be identified or tracked. This is just a basic way to gather operating system data and rate of application usage (startup only). The public client IP address is of course known by the server (just like any other HTTP request made by any other service on the web) but the EDRLab server doesn't store this information in its telemetry database.

**Comment by danielweck** [comment]
Client code:
https://github.com/edrlab/thorium-reader/blob/0cde7e3c45c19383c03a06134dc82af4b240aac4/src/main/redux/sagas/telemetry.ts#L82-L90

**Comment by danielweck** [comment]
Server code:
```
INSERT INTO logs (os_version, locale, os_ts, fresh_install, entry_type, current_version, prev_version, new_install)
```
https://github.com/edrlab/thorium-telemetry-server/blob/8878994993af1cad0251c21286f3e0ab62873572/lua/script.template.lua#L174

**Comment by danielweck** [comment]
The Thorium dev team will discuss the "opt out" feature you are suggesting.
In the meantime, if you are still concerned about your privacy with Thorium's minimalist telemetry, you can block HTTP requests to https://telemetry.edrlab.org

**Comment by danielweck** [comment]
bug triage: moving to discussion
